src/hypothesis/extra/typehints.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import print_function
+import sys
+from hypothesis import strategies as st
+from hypothesis.strategies import SearchStrategy
+from typing import Dict, Tuple, List, Iterator, Set, Union, Optional, NamedTuple, Callable, Sequence, Iterable, Generator
+import operator
+from collections import OrderedDict
+compose = lambda f,g: lambda *x: f(g(*x))
+# see https://docs.python.org/3/library/typing.html
+try:
+    unicode = unicode
+except:
+    unicode = str
+    from functools import reduce
+
+primitives = {
+    str   : st.text(), # might want to default to `string.printable`
```

Comment:
Oh, my understanding of the difference between `st.characters` and `st.text` is incorrect. It looks like the only difference is the parameters they take?